### DISCUSSION TITLE
İbax puan sistemi

### DISCUSSION THREAD

**Discussion by Trajedi032** [IBAX-io/go-ibax#1475] [Ideas]
Bu konu hakkında yardımcı olabilecek var mı?

**Comment by Skips1** [comment]
Tam olarak bende bilmiyorum kanka :(

**Comment by atakanfazli** [comment]
Retweetler, buraya yazılan yorumlar, github da star ve fork puanları gibi birçok puan kasma yöntemi var. En çok puan retweetten geliyor bildiğim kadarıyla bir de referanslar çok önemli
